saos-webapp/src/main/webapp/WEB-INF/view/template/pageDescription.jsp

```diff
@@ -1,13 +0,0 @@
-<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
-
-<tiles:importAttribute name="metaPageDescriptionCode" />
-
-<%-- Page description only for the root of the given page, for example for /search but not for /search?dateFrom... 
-     Search engines usually omit page descriptions if they are the same for many pages 
---%>
-<c:if test="${empty pageContext.request.parameterMap}"> 
-	<c:if test="${!empty metaPageDescriptionCode}">
-	        <meta name="description" content="<spring:message code='${metaPageDescriptionCode}'/>"/>
-	</c:if>
-</c:if>
```

Comment:
I've thought about creating description for search (in the future), that contains serach parameters, so it would make it not the same for different pages.
For now I'll leave old version with pageDescription.
:ok_hand: